From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 18 May 2019 12:20:34 +0200
Subject: [PATCH] {BUG in 5.2-rc1} include/linux/mfd/stmfx.h: fix header guard

clang warns:

    include/linux/mfd/stmfx.h:7:9: error: 'MFD_STMFX_H' is used as a
    header guard here, followed by #define of a different macro
          [-Werror,-Wheader-guard]
    #ifndef MFD_STMFX_H
            ^~~~~~~~~~~
    include/linux/mfd/stmfx.h:8:9: note: 'MFX_STMFX_H' is defined here;
    did you mean 'MFD_STMFX_H'?
    #define MFX_STMFX_H
            ^~~~~~~~~~~
            MFD_STMFX_H
---
 include/linux/mfd/stmfx.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/mfd/stmfx.h b/include/linux/mfd/stmfx.h
index d890595b89b6..3c67983678ec 100644
--- a/include/linux/mfd/stmfx.h
+++ b/include/linux/mfd/stmfx.h
@@ -5,7 +5,7 @@
  */
 
 #ifndef MFD_STMFX_H
-#define MFX_STMFX_H
+#define MFD_STMFX_H
 
 #include <linux/regmap.h>
 
-- 
